docs/source/examples/workspaces/libvirt/PinFile

```diff
@@ -25,6 +25,7 @@ libvirt-new:
               - name: default
               - name: default
                 dhcp: false
+                static_ip_reserved: false
```

Comment:
Better have a new example, with two networks (one should have static_ip enabled)